backend/api/visualization.proto

```diff
@@ -0,0 +1,120 @@
+// Copyright 2019 Google LLC
```

Comment:
how does the output(artifact path to the HTML file) get passed back to the client?